addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -95,8 +95,8 @@ func _ready():
 		
 		# Connect to singleton signals
 		E.language_changed.connect(_translate)
-		G.blocked.connect(_on_graphic_interface_blocked)
-		G.unblocked.connect(_on_graphic_interface_unblocked)
+		#G.blocked.connect(_on_graphic_interface_blocked)
```

Comment:
If you think those lines are useless now, I'd clean those up.

Reply:
Yes. I always forget to remove commented lines. Thanks!